container/values.yml

```diff
@@ -9,21 +11,22 @@ env:
 
   # --- KEG-CLI ENV CONTEXT --- #
 
-  # Set the paths to the linked external app
-  # The app should be linked to the keg-cli with `kee`
-  # Example command to link the app => `keg tap link kee`
+  # Docker / Docker Compose paths
   KEG_DOCKER_FILE: "{{ cli.taps.links.consumer }}/container/Dockerfile"
   KEG_VALUES_FILE: "{{ cli.taps.links.consumer }}/container/values.yml"
+
```

Comment:
Not something you did, but all these `cli.taps.links.consumer` should be updated to `cli.taps.consumer.path` 

The globalConfig was updated a little bit ago, and I think this repo was forgotten about. 
Can you update these so they point to the correct locations. For example =>

WRONG
`KEG_DOCKER_FILE: "{{ cli.taps.links.consumer }}/container/Dockerfile"`

CORRECT
`KEG_DOCKER_FILE: "{{ cli.taps.consumer.path }}/container/Dockerfile"`

Reply:
updated